lint/doc/Developing.md

```diff
@@ -0,0 +1,28 @@
+# Witnesslint Development Instructions
+
+## Installation for Development
+
+0. You will need a python version >=3.5.
+
+1. Witnesslint uses `poetry` for dependency management and packaging.
+Install poetry with `pip install poetry`.
```

Comment:
Does this command make sense? (Not sure because I don't know Poetry.)

Using `pip install` would typically require to be currently using a virtual environment and install Poetry in this virtual environment. But Poetry is supposed to manage virtual environments on its own for the project, so I am not sure whether using a virtual environment is a problem?

And outside of a virtual environment `pip install` without `--user` won't work.

But in general we can probably also avoid telling people how to install Poetry and simply refer to its docs?

Reply:
True, replaced that line with a link to the official installation instructions.